packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -79,6 +79,10 @@ const propTypes = {
    * Ref callback function to be applied to content container.
    */
   refCallback: PropTypes.func,
+  /**
+   * Header Title for menu if menu contains drill-in sub-menus
```

Comment:
I though this was specific to the root menu, not sub menus?

Reply:
Updated here 277939ef97fccdb445d4a11850b83f430e56c73b